src/Entity/AppInfo.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\AppInfoRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: AppInfoRepository::class)]
+class AppInfo
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(nullable: false)]
+    private ?int $id = null;
+
+    #[ORM\Column(length: 255, nullable: false)]
+    private ?string $makairaDomain = null;
```

Comment:
I still don't get, why this Column is not nullable, but you set the dafult Value to null? Isn't that contradictory? If you want an empty default value, wouldn't an empty string be better?